dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,414 @@
+
+### Video
+
+For an example of an instream video ad unit, see below.  For more detailed instructions, see [Show Video Ads]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
+
+```javascript
+pbjs.addAdUnits({
+    code: 'video',
+    mediaTypes: {
+        video: {
+            context: "instream",
+            sizes: [640, 480],
+        },
+    },
+    bids: [{
+        bidder: 'appnexus',
+        params: {
+            placementId: '9333431',
+            video: {
+                skippable: true,
+                playback_methods: ['auto_play_sound_off']
+            }
+        }
+    }]
+});
+```
+
+For an example of an outstream video ad unit, see below.  For more detailed instructions, see [Show Outstream Video Ads]({{site.baseurl}}/dev-docs/show-outstream-video-ads.html).
+
+```javascript
+pbjs.addAdUnit({
+    code: 'video1',
+    mediaTypes: {
+        video: {
+            context: 'outstream',
+            sizes: [640, 480]
+        }
+    },
+    renderer: {
+        url: 'http://cdn.adnxs.com/renderer/video/ANOutstreamVideo.js',
+        render: function(bid) {
+            ANOutstreamVideo.renderAd({
+                targetId: bid.adUnitCode,
+                adResponse: bid.adResponse,
+            });
+        }
+    },
+    ...
+})
+```
+
+<a name="adUnit-native-example">
+
+### Native
+
+For an example of a native ad unit, see below.  For more detailed instructions, see [Show Native Ads]({{site.baseurl}}/dev-docs/show-native-ads.html).
+
+```javascript
+pbjs.addAdUnits({
+    code: slot.code,
+    mediaTypes: {
+        native: {
+            image: {
+                required: true,
+                sizes: [150, 50]
+            },
+            title: {
+                required: true,
+                len: 80
+            },
+            sponsoredBy: {
+                required: true
+            },
+            clickUrl: {
+                required: true
+            },
+            body: {
+                required: true
+            },
+            icon: {
+                required: true,
+                sizes: [50, 50]
+            },
+        },
+        bids: [{
+            bidder: 'appnexus',
+            params: {
+                placementId: '9880618'
+            }
+        }, ]
+    }
+})
+```
+
+<a name="adUnit-multi-format-example">
+
+### Multi-Format
+
+For an example of a multi-format ad unit, see below.  For more detailed instructions, see [Show Multi-Format Ads]({{site.baseurl}}/dev-docs/show-multi-format-ads.html).
+
+{% highlight js %}
+
+    pbjs.addAdUnits([{
+            code: 'div-banner-native',
+            mediaTypes: {
+                banner: {
+                    sizes: [
+                        [300, 250]
+                    ]
+                },
+                native: {
+                    type: 'image'
+                },
+            },
+            bids: [{
+                bidder: 'appnexus',
+                params: {
+                    placementId: 12340414,
```

Comment:
Here, "placementId" is being passed as an int, but above it's being passed as a String.  Let's make sure we're consistent across examples.

Reply:
Thanks - updated in 42b9c7d to pass as string, since that's also what we show in the AN adapter example on the [Bidder params page](http://prebid.org/dev-docs/bidders.html#appnexus)